<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlueprintUI</class>
 <widget class="QDialog" name="BlueprintUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Blueprints</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QPushButton" name="duplicateBlueprintButton">
     <property name="text">
      <string>Duplicate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="newEntityButton">
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="7">
    <widget class="QListWidget" name="entityList"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="editEntityButton">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="deleteEntityButton">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" rowspan="2">
    <widget class="QPushButton" name="closeWindowButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeWindowButton</sender>
   <signal>pressed()</signal>
   <receiver>BlueprintUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
